### ISSUE TITLE
Feature request - Streamer.bot client

### ISSUE THREAD

**Issue by rondhi** [izzy/stream-chat#53]
I noticed that you're working on a refactor of the code. Any plans on implementing Streamer.bot client library over the Websocket client?

https://streamerbot.github.io/client/

**Comment by izzy** [comment]
TL;DR: Thanks for the notice but not planning on using it.

The long explanation:

I did not know this was a thing but after just looking at the license it's sadly a no-go for me personally.

If I ever get to finishing it it'll be a rewrite, not just a refactor. The current code is too much of an organically grown mess. I count that as lessons learned but not technical debt I want to move forward. But honestly, my personal excitement for streaming itself has faded years ago, so this is a pure coding exercise for me, with little to know personal investment, even more so since I switched away from SB for almost all things a long time ago.

The rewrite will be GPL3+ and while using MIT in GPL code is possibly permissible(although IANAL) I don't have the energy to deal with permissive licenses like that anymore. I'm very opinionated about FOSS, which is one of the main reasons I don't use SB anymore.

All that said, just from the documentation it doesn't even seem that much of a help, especially since I'd then have to write another wrapper around that wrapper since the plan was to rewrite the code in such a way that the overlay can be extended to be used with or without Streamer.Bot's APIs(i.e. with another Bot, like one that's not closed source).
